codalab/rest/schemas.py

```diff
@@ -221,6 +221,25 @@ class Meta:
         type_ = 'bundles'
 
 
+class BundleLocationSchema(Schema):
+    bundle_uuid = fields.String(validate=validate_uuid, attribute='uuid')
+    bundle_store_uuid = fields.String(validate=validate_uuid, attribute='uuid')
+
+    class Meta:
+        type_ = 'bundle-location'
+
+
+class BundleLocationListSchema(Schema):
+    # Used only for bundle locations GET endpoint
```

Comment:
Maybe remove this comment? (because this is also used in the add bundle locations POST endpoint)